modules/custom/dumpsoftwareversions/main.nf

```diff
@@ -100,7 +100,7 @@ process CUSTOM_DUMPSOFTWAREVERSIONS {
     with open("software_versions_mqc.yml", 'w') as f:
         yaml.dump(versions_mqc, f, default_flow_style=False)
 
-    with open('versions.yml', 'w') as f:
+    with open('versions.out.yml', 'w') as f:
         yaml.dump(module_versions, f, default_flow_style=False)
```

Comment:
```suggestion
```

Reply:
This was already in the PR, the only thing missing was the change in the test, let's see if tests work now...